docs/install.md

```diff
@@ -61,13 +62,39 @@ Actually we use vanilla free5gc::docker-compose with some overrides(see docker-c
 
 So you can clone free5gc docker-compose and test free5gc upf, then add edgecom override files and feel the differences.
 
-0. pull repository: `git clone https://github.com/edgecomllc/free5gc-compose.git`
+0. Pull repository: `git clone https://github.com/edgecomllc/free5gc-compose.git`
 0. Run containers based on docker hub images:
    ```bash
    cd free5gc-compose
    docker compose pull
    docker compose up -d
    ```
+0. Check if UPF and SMF containers are running correctly:
+   ```bash
+   sudo docker compose logs edgecom-upf
+   ```
+   ```bash
+   sudo docker compose logs free5gc-smf
+   ```
+0. Before running UE emulator you have to register UE in 5G core network using web console:
+   - Type `localhost:5000/#/` in the address bar of your browser. 
+    - Username: admin Password: free5gc
+    - In the console click on "Subscribers" section. In that section click on "New Subcriber" button.
+   - IMSI have to be the same as in UE config.
+   - Click "Submit" button.
+0. The command that launches UE emulator has to be executed inside of ueransim container:
+   ```bash
+   sudo docker compose exec ueransim bash
+   ```
+0. Now UE emulator can be launched
```

Comment:
During execution of this command UE should get IP address. This can be verified in UE output